README.md

```diff
@@ -12,24 +12,23 @@
 
 ## Project
 
-Wasabi A/B Testing Service is a real-time, enterprise-grade, 100% API driven project. Users are empowered to own their own data, and run experiments across web, mobile, and desktop. It’s fast, easy to use, it’s chock full of features, and instrumentation is minimal.
-
-Learn more about how Wasabi can empower your team to move from hunches to actionable, data-driven user insights with our simple, flexible, and scalable experimentation platform.
+Wasabi is a mature, feature-rich, open-source A/B testing platform, complete with an experiment management UI. Wasabi powers all of Intuit's in-product experimentation and has been running in production at scale for several years.
 
+Wasabi differs from other A/B testing services and frameworks in two ways. First, Wasabi runs on your servers, giving you full control over your data. Second, as something we call _A/B-Testing-as-a-Service_, Wasabi focuses on API-level integration with your apps and products. This means that Wasabi can be used to run experiments across Web, mobile, and desktop user experiences, as well as experiements on back-end, non-user-facing components, like business process workflows, search and routing algorithms, and predictive models.
 
 ### Features
 
-* **Own your own data** - Don’t ship your data to 3rd party services. Manage your own data and protect your users to maintain your competitive advantage.
-* **Enterprise Grade** - Wasabi has been battle tested in production in fin-tech for 2 years in TurboTax, QuickBooks, Mint.com, etc., powering over 1,000 experiments across Intuit.
-* **High Performance** - Consistent performance response times within 30ms server side.
-* **100% API Driven** - Compatible with any language and environment.
-* **Platform Agnostic** - A uniform, consistent experience across web, mobile, desktop, and also front-end, back-end.
-* **Get up and running in minutes** - Spin up your Wasabi docker in 5 minutes and be in production with the platform, instrumentation, and experiments within a day.
-* **It’s Free** - Don’t pay an arm and a leg for an enterprise-grade platform – the platform is free! 
-* **Real-time assignments** - Assign users into experiments in real-time, to preserve traffic for other parallel A/B tests.
+* **Own your own data** - Wasabi runs on your servers, in the cloud or on-premise, where you have complete control over your data.
```

Comment:
I like highlighting Wasabi being free but it is implied as it is open source and am ok with removing it.